.circleci/config.yml

```diff
@@ -73,8 +75,9 @@ references:
     DO_TF_WORKSPACE: dev
     RELEASE_NAME: codacy-release
     NAMESPACE: codacy-release
-    K8S_VERSION: "1.15"
-    NUM_NODES: "5"
+    ENVIRONMENT: release
```

Comment:
Used for pattern-matching with the templated values-db-<environment>.yaml file created by terraform